packages/amplify-codegen-appsync-model-plugin/src/visitors/appsync-swift-visitor.ts

```diff
@@ -304,4 +307,39 @@ export class AppSyncSwiftVisitor extends AppSyncModelVisitor {
     }
     return !field.isNullable;
   }
+
+  protected generateAuthRules(model: CodeGenModel): string {
+    const authDirectives: AuthDirective[] = (model.directives.filter(d => d.name === 'auth') as any) as AuthDirective[];
+    const rules: string[] = [];
+    authDirectives.forEach(directive => {
+      directive.arguments?.rules.forEach(rule => {
+        const authRule = [];
+        switch (rule.allow) {
+          case AuthStrategy.owner:
+            authRule.push('allow: .owner');
+            authRule.push(`identityClaim: "${rule.identityClaim}"`);
+            authRule.push(`ownerField: "${rule.ownerField}"`);
```

Comment:
can we reoder this to match the constructor's ordering of parameters?
```
authRule.push('allow: .owner');
authRule.push(`ownerField: "${rule.ownerField}"`);
authRule.push(`identityClaim: "${rule.identityClaim}"`);
```

Reply:
here's the ordering https://github.com/aws-amplify/amplify-ios/blob/c9ab8618b5f74ce979c7e2cc392e3bfd75918858/Amplify/Categories/DataStore/Model/Schema/Model%2BSchema.swift#L48